docs/source/en/using-diffusers/workflows.md

```diff
@@ -0,0 +1,331 @@
+             ('beta_start', 0.00085),
+             ('beta_end', 0.012),
+             ('beta_schedule', 'scaled_linear'),
+             ('trained_betas', None),
+             ('skip_prk_steps', True),
+             ('set_alpha_to_one', False),
+             ('prediction_type', 'epsilon'),
+             ('timestep_spacing', 'leading'),
+             ('steps_offset', 1),
+             ('_use_default_values', ['prediction_type', 'timestep_spacing']),
+             ('_class_name', 'PNDMScheduler'),
+             ('_diffusers_version', '0.6.0'),
+             ('clip_sample', False)])}
+```
+
+A [`Workflow`] object provides all the argument values in the `__call__()` of a pipeline. Add `return_workflow=True` to return a `Workflow` object:
+
+Once you have generated a workflow object, you can serialize it with [`~Workflow.save_workflow`]:
+
+```python
+outputs.workflow.save_workflow("my-simple-workflow-sd")
+```
+
+By default, your workflows are saved as `diffusion_workflow.json`, but you can give them a specific name with the `filename` argument:
+
+```python
+outputs.workflow.save_workflow("my-simple-workflow-sd", filename="my_workflow.json")
+```
+
+You can also set `push_to_hub=True` in [`~Workflow.save_workflow`] to directly push the workflow object to the Hub. 
+
+## Loading a workflow
+
+You can load a workflow in a pipeline with [`~DiffusionPipeline.load_workflow`]:
+
+```python
+from diffusers import DiffusionPipeline
+import torch
+
+pipeline = DiffusionPipeline.from_pretrained(
+    "runwayml/stable-diffusion-v1-5", torch_dtype=torch.float16
+).to("cuda")
+
+pipeline.load_workflow("sayakpaul/my-simple-workflow-sd")
+```
+
+Once the pipeline is loaded with the desired workflow, it's ready to be called:
+
+```python
+image = pipeline().images[0]
+```
+
+By default, while loading a workflow, we don't modify the scheduler of the underlying pipeline from the workflow. But you can change this behaviour by doing:
+
+```
+pipeline.load_workflow("sayakpaul/my-simple-workflow-sd", load_scheduler=True)
+```
+
+This is particularly useful if you had changed the scheduler after loading a pipeline.
+
+You can also override the pipeline call arguments. For example, to add a `negative_prompt`:
+
+```python
+image = pipeline(negative_prompt="bad quality").images[0]
+```
+
+Loading from a workflow is possible by specifying the `filename` argument inside the [`DiffusionPipeline.load_workflow`] method.
+
+A workflow doesn't necessarily have to be used with the same pipeline that generated it. You can use it with a different pipeline too:
+
+```python
+from diffusers import DiffusionPipeline
+import torch
+
+pipeline = DiffusionPipeline.from_pretrained(
+    "stabilityai/stable-diffusion-xl-base-1.0", torch_dtype=torch.float16
+).to("cuda")
+
+pipeline.load_workflow("sayakpaul/my-simple-workflow-sd")
+image = pipeline().images[0]
+```
+
+However, make sure to thoroughly inspect the values you are calling the pipeline with, in this case.
```

Comment:
Is this what you mean?

```suggestion
However, make sure the workflow values from the old pipeline are compatible with the new pipeline you're using.
```

Reply:
Not just compatibility. If you're using a workflow that was generated using SD in SDXL, it's recommended to check the settings since the default SD height and width might not play nicely with SDXL. So, I think "inspection" is a better word here.